### PR TITLE
Create multi-module project setup for HBase 1.x and HBase 2.x

### PR DESCRIPTION
**Pull request by akshayarni**
Adding demo for tests using the `DockerComposeContainer` for HBase 1.x version.

### CODE REVIEW

File: junit-testcontainer-hbase-1.x/src/test/resources/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: '2'
+services:
+  hbase:
+    image: akshayarni/testhbase1.1.2:v12
```
Comment (Tushar-Naik) [inline junit-testcontainer-hbase-1.x/src/test/resources/docker-compose.yml]: this will be changed?

File: junit-testcontainer-hbase-1.x/pom.xml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>dns-cache-manipulator</artifactId>
+            <version>${dns.cache.manipulator.version}</version>
```
Comment (Tushar-Naik) [inline junit-testcontainer-hbase-1.x/pom.xml:85]: test scope can be added here

File: junit-testcontainer-hbase-1.x/pom.xml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        <dependency>
+            <groupId>io.appform.testcontainer</groupId>
+            <artifactId>junit-testcontainer-commons</artifactId>
+            <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>junit</groupId>
+                    <artifactId>junit</artifactId>
+                </exclusion>
+            </exclusions>
```
Comment (Tushar-Naik) [inline junit-testcontainer-hbase-1.x/pom.xml:38]: test scope can be added here

**Review by Tushar-Naik — COMMENTED** [review]
there seems to be an empty .gitignore file

File: junit-testcontainer-hbase-1.x/src/test/resources/docker-compose.yml (hunk 1 above)
Comment (akshayarni) [reply junit-testcontainer-hbase-1.x/src/test/resources/docker-compose.yml]: Changed to a different publicly available 1.3.2 version image.

File: junit-testcontainer-hbase-1.x/pom.xml (hunk 2 above)
Comment (akshayarni) [reply junit-testcontainer-hbase-1.x/pom.xml:85]: done.

File: junit-testcontainer-hbase-1.x/pom.xml (hunk 3 above)
Comment (akshayarni) [reply junit-testcontainer-hbase-1.x/pom.xml:38]: done

**Review by Tushar-Naik — COMMENTED** [review]
LTGM

**Pull request merged**